main.py

```diff
@@ -1,7 +1,53 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    genre_to_create = ["Western", "Action", "Dramma"]
+
+    actors_to_create = [("George", "Klooney"),
+                        ("Kianu", "Reaves"),
```

Comment:
The name 'Kianu Reaves' seems to be a typo. It is later updated to 'Keanu Reeves', which suggests that 'Keanu Reeves' should be used initially.